### ISSUE TITLE
Colors of graph

### ISSUE THREAD

**Issue by baqueroefren** [ChristopherChudzicki/math3d-react#356]
Good morning.
Is it possible to graph a vector field on a surface?
because all the objects that I add remain the same color?
how is this controlled?

**Comment by ChristopherChudzicki** [comment]
Good idea! That is not currently possible. I'm currently working on a re-write of math3d, and have opened https://github.com/ChristopherChudzicki/math3d-next/issues/375 to track this.

**Comment by ChristopherChudzicki** [comment]
I'm not sure I understand your color question. Try click-and-hold on the color dot to change colors. (Which...needs to be a more discoverable feature.)
